Example/Podfile

```diff
@@ -50,13 +51,19 @@ target 'Emission' do
   pod 'Artsy+Authentication', :subspecs => %w(email),
                               :git => 'https://github.com/artsy/Artsy-Authentication.git',
                               :branch => 'fetch-user-details'
+  # Stripe- tag must match package.json's tipsi-stripe dep
+  pod 'tipsi-stripe',
+    :git => "https://github.com/erikdstock/tipsi-stripe.git",
+    :branch => 'fix-podspec-requires-packagejson'
+    #:podspec => tispi_stripe_spec_path
```

Comment:
Can we remove this line?

Reply:
I update this to remove the `git/branch` keys and use `podspec:` again, instead keeping our dependency on only the single fork in `package.json` ([51e1a1f](https://github.com/artsy/emission/pull/1086/commits/51e1a1f6bdce3f5463d89053a1d518fd0b573f21))